i18n/locales/index.mjs

```diff
@@ -0,0 +1,52 @@
+// Next.js React Intl Translation Manifest
+// In order to add new Translations to this application, please add the new locale
+// to the `i18n/config.json` file and then add the new locale file to this directory
+// and import it below.
+
```

Comment:
That not increase bundle size ?

Reply:
It does increase the initial bundle size, even tho code-splitting will ensure this is loaded at the right moment.

But ultimately, the page bundle will still be lower since we're not setting the i18n data on every page static `pageProps`, which usually gets both imprinted on the HTML and on a separate JSON file that Next.js loads.

So, the HTML bundle decreases, but the JavaScript one will increase slightly a bit, which is barely not-noticeable when gzipping the request. So it is a valid tradeoff.

Reply:
Perfect! Just you would be given figures?

Reply:
I will do that and give you diff

Reply:
I don't need a diff, Augustin